src/BuildingRegistry.Projector/Consumer/ConsumerController.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace BuildingRegistry.Projector.Consumer
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.Api;
+    using Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Consumer;
+    using BuildingRegistry.Consumer.Address;
+    using BuildingRegistry.Infrastructure;
+    using Dapper;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Data.SqlClient;
+    using Microsoft.Extensions.Configuration;
+
+    [ApiVersion("1.0")]
+    [ApiRoute("consumers")]
+    public class ConsumersController : ApiController
+    {
+        private const string? ConsumerConnectionStringKey = "ConsumerAddress";
+
+        [HttpGet]
+        public async Task<IActionResult> Get(
+            [FromServices] IConfiguration configuration,
+            CancellationToken cancellationToken = default)
+        {
+            await using var sqlConnection =
+                new SqlConnection(configuration.GetConnectionString(ConsumerConnectionStringKey));
+            var result =
+                await sqlConnection.QueryFirstAsync<ProcessedMessage>(
+                    $"SELECT * FROM [{Schema.ConsumerAddress}].[{AddressConsumerItemConfiguration.TableName}] ORDER BY [{nameof(ProcessedMessage.DateProcessed)}] DESC");
```

Comment:
`[{AddressConsumerItemConfiguration.TableName}]` is the wrong table. Check `ProcessedMessages`